scripts/prep-spd-roster.py

```diff
@@ -0,0 +1,54 @@
+import pandas as pd
+import sys
+
+from pandas.core.frame import DataFrame
+
+
+def main():
+    date = sys.argv[1]
+    in_csv = sys.argv[2]
+    out_csv = sys.argv[3]

```

Comment:
Just a thought, the `click` library might be super useful and simple for this case

Reply:
I was gonna look at that and then I didn't. I should though you're right.